src/IRtext.cpp

```diff
@@ -135,6 +135,7 @@ const PROGMEM char* kEyeAutoStr = D_STR_EYEAUTO;
 const PROGMEM char* kLightToggleStr = D_STR_LIGHTTOGGLE;
 const PROGMEM char* kOutsideQuietStr = D_STR_OUTSIDEQUIET;
 const PROGMEM char* kPowerToggleStr = D_STR_POWERTOGGLE;
+const PROGMEM char* kPowerButtonStr = D_STR_POWERBUTTON;
```

Comment:
@crankyoldgit just want to ping you about this "button" string has been added

Reply:
Yep. LGTM. That's the correct way to do it.